### PR TITLE
Update time range preset values mnrva 799

### PR DESCRIPTION
**Pull request by pras7430**
## JIRA:
See ticket here - https://jira.rax.io/browse/MNRVA-799

 ## Description:

1. Time range component update and added separately at visualize component
2. test case corrected.
3. Query parameter correction on the selection of filters.
4.
 ## Testing:

 ## Screenshots:
Screenshot 2021-06-22 at 1 52 48 PM

(if applicable)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -93,7 +93,7 @@
     "karma-coverage-istanbul-reporter": "2.0.1",
     "karma-jasmine": "3.0.1",
     "karma-jasmine-html-reporter": "1.4.2",
-    "protractor": "7.0.0",
+    "protractor": "^7.0.0",
```
Comment (lovedeep92) [inline package.json:96]: can you please remove ^

Comment (FarajiA) [inline package.json:96]: Not sure if this ^ was purposely done

**Review by FarajiA — APPROVED** [review]
LGTM

File: package.json (hunk 1 above)
Comment (pras7430) [reply package.json:96]: good catch, it was just experimental

Comment (pras7430) [reply package.json:96]: 👍🏻

**Pull request merged**